### ISSUE TITLE
[COGNITION]: Alert Box - Icons SHOULD NOT be invisible tab stops for screen readers

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9028]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our AlertBox component includes an optional [close button that appears as a FontAwesome icon](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/8fcb419ba9dab1a3d242fd53f539c3ab2a5e0460/packages/formation-react/src/components/AlertBox/AlertBox.jsx#L52). The button has an `aria-label` associated with it, so the icon should be hidden from screen readers. JAWS tends to make these icons a silent "tab stop" which can be confusing to users. Code snippet below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

**As a screen reader user, I want the following to be true:**
- [ ] I hear "Close notification" announced
- [ ] I do not have to press `DOWN_ARROW` twice to get to the next text element. As long as the `<i>` is hidden properly, users will only have to press the arrow key one time.

**As a visual user, I want the following to be true:**
- [ ] Visual styling should not change

### Existing Code

```diff
- <i className="fas fa-times-circle" aria-label="Close icon" />
```

### Proposed Code
```diff
+ <i aria-hidden="true" className="fas fa-times-circle" role="presentation" />
```

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #242](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/242) via [**ZenHub**](https://www.zenhub.com/)
